### PR TITLE
cpack configs for jpscore

### PR DESCRIPTION
**Pull request by chraibi**

### CODE REVIEW

**Comment by chraibi** [comment]
I don't like it making a bin directory. Kai: any idea?

**Comment by Ozaq** [comment]
No sorry, didn't look into mac packaging so far.

**Comment by chraibi** [comment]
@Ozaq I changes the purpose of this PR. It should be concerned with CPack in general. 
Now checking for osx, later for windows and ubuntu. 

But, if you have time, could you check on the refactoring commit?

File: cmake_modules/CPackUtilities.cmake [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  list(APPEND CPACK_GENERATOR "NSIS")
+  set (CPACK_GENERATOR  ${CPACK_GENERATOR} PARENT_SCOPE)
+  find_program(NSIS_PATH nsis PATH_SUFFIXES nsis)
+  if(NSIS_PATH)
+    message(STATUS "   + NSIS                                 YES ")
+  else(NSIS_PATH)
+    message(STATUS "   + NSIS                                 NO ")
+  endif(NSIS_PATH)
+
+  #set(CMAKE_INSTALL_MFC_LIBRARIES ON)
```
Comment (Ozaq) [inline cmake_modules/CPackUtilities.cmake]: commented out code :) this can be removed

File: cmake_modules/CPackUtilities.cmake [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP TRUE)
+  set(VCPKG_DIR "${_VCPKG_INSTALLED_DIR}/${VCPKG_TARGET_TRIPLET}/bin/")
+  set(DIRS ${VCPKG_DIR})
+  #file(GLOB DLL_FILES "${VCPKG_DIR}/*.dll")
```
Comment (Ozaq) [inline cmake_modules/CPackUtilities.cmake]: commented out code :) this can be removed

Comment (chraibi) [reply cmake_modules/CPackUtilities.cmake]: done

File: cmake_modules/CPackUtilities.cmake (hunk 1 above)
Comment (chraibi) [reply cmake_modules/CPackUtilities.cmake]: done

**Comment by chraibi** [comment]
thanks for your review @Ozaq

**Pull request merged**